packages/dds/tree/src/test/memory/.mocharc.cjs

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file for memory profiling tests
+ */
+
+const getFluidTreeTestMochaConfig = require("../../../.mocharc.cjs");
+
+const extendedConfig = {
+	...getFluidTreeTestMochaConfig,
+	"fgrep": ["@Benchmark", "@MemoryUsage"],
+	"node-option": [
```

Comment:
You could consider augmenting any node options in the base config with expose-gc and gc-global rather than redefining all the flags. If you want to do this, i'd make sure the config works when no node-option is specified in the base config.

Reply:
`node-option` can either be a [string](https://github.com/microsoft/FluidFramework/blob/cb8a88bb8efce6bc37e7079329eaacfa474c80bc/packages/test/mocha-test-setup/mocharc-common.cjs#L54) or an [array of strings](https://github.com/microsoft/FluidFramework/blob/cb8a88bb8efce6bc37e7079329eaacfa474c80bc/packages/dds/map/src/test/memory/.mocharc.cjs#L13). 

I added a ternary operator to enable augmentation.